wrc20.md

```diff
@@ -255,10 +254,9 @@ A module of shorthand commands for the WRC20 module.
       )
 
       #wrc20ReverseBytes
-      [macro]
 
-    rule #wrc20ReverseBytesTypeIdx => 1 [macro]
-    rule #wrc20ReverseBytesType    => [ i64 ] -> [ i64 ] [macro]
+    rule #wrc20ReverseBytesTypeIdx => 1
+    rule #wrc20ReverseBytesType    => [ i64 ] -> [ i64 ]
```

Comment:
@radumereuta I'm trying to remove the macro label from this line and it's causing Kprove to parse the rest of this file as a part of this rule. If I add an attribute like [color(red)] and move the macro to the syntax, it ends up working. If I simply move the macro to the production, I get this output:
```
+ make test-prove -j2

./kwasm prove --backend haskell tests/proofs/memory-spec.k KWASM-LEMMAS --format-failures \
--concrete-rules WASM-DATA.wrap-Positive,WASM-DATA.setRange-Positive,WASM-DATA.getRange-Positive
./kwasm prove --backend haskell tests/proofs/functions-spec.k FUNCTIONS-LEMMAS --format-failures \
--concrete-rules WASM-DATA.wrap-Positive,WASM-DATA.setRange-Positive,WASM-DATA.getRange-Positive

#Top

./kwasm prove --backend haskell tests/proofs/loops-spec.k KWASM-LEMMAS --format-failures \

#Top

./kwasm prove --backend haskell tests/proofs/wrc20-spec.k WRC20-LEMMAS --format-failures \
--concrete-rules WASM-DATA.wrap-Positive,WASM-DATA.setRange-Positive,WASM-DATA.getRange-Positive,WASM-DATA.get-Existing,WASM-DATA.set-Extend

[Error] Inner Parser: Parse error: unexpected end of file following token '->'.
	Source(/home/jenkins-slave/workspace/KWasm_PR-445@2/wrc20.md)
	Location(259,49,259,50)

[Error] Compiler: Had 1 parsing errors.
Makefile:166: recipe for target 'tests/proofs/wrc20-spec.k.prove' failed
make: *** [tests/proofs/wrc20-spec.k.prove] Error 113
make: *** Waiting for unfinished jobs....

script returned exit code 2
```

Do you have any ideas on how I could get around this error? Adding `[color(red)]` label is definitely no the right way..

Reply:
No, that's the right solution.
The outer parser tries to parse attributes at the end of the rule, and it prefers that branch.
You could say `rule A => B [ i64 ] [dummyAtt]`